### ISSUE TITLE
Release Request for v0.55

### ISSUE THREAD

**Issue by phil-airship** [zappa/Zappa#1104]
Specifically looking for a release which includes
https://github.com/zappa/Zappa/pull/1083

This fixes a compatibility issue which currently prevents newer version of specific libraries from being used.
Cryptography being a common and notable case.
It is worth getting addressed before more libraries with linux-specific native code move away from older
versions of manylinux

**Comment by NinitoAS** [comment]
There are quite a few awesome changes in Zappa which would be perfect to have in a 0.55 release

**Comment by phil-airship** [comment]
@hellno @javulticat Wanted to ping yall on this since you commented on the PR that I had initially made

**Comment by javulticat** [comment]
Thanks for the bump @phil-airship.  We are currently working through a bug with our CI/CD process and how it publishes releases to `pypi`.  We'll need to merge #1103 first, and then we'll publish a new release shortly thereafter.  Thanks for your patience!

**Comment by phil-airship** [comment]
Wanted to follow up on this since it is an update that will unblock dependency limits and the other PR this was waiting on seems stalled

**Comment by monkut** [comment]
duplicate of https://github.com/zappa/Zappa/pull/1083

closing (planned for next release).

**Comment by phil-airship** [comment]
Hey @monkut I saw that this was closed and the new release was cut, but the release did not include the master branch changes which have landed since last November https://github.com/zappa/Zappa/compare/0.54.2...master

**Comment by monkut** [comment]
@phil-airship Yes, 0.54.2 was a documention only releas.

The recently releaseed 0.55.0 contains this pr.
